indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3271,7 +3271,9 @@ private void checkPendingCompletionTasks()
             log.warn("All tasks in group [%d] failed to publish, killing all tasks for these partitions", groupId);
           } else {
             log.makeAlert(
-                "No task in [%s] for taskGroup [%d] succeeded before the completion timeout elapsed [%s]!",
+                "No task in [%s] for taskGroup [%d] succeeded before the completion timeout elapsed [%s]! "
+                + "Check metrics and logs to see if the creation, publish or handoff of any segment is taking longer "
+                + "than usual.",
```

Comment:
Nit: break the line sooner
```suggestion
                + "Check metrics and logs to see if the creation, publish or handoff"
                + "  of any segment is taking longer than usual.",
```